### ISSUE TITLE
Conduct content audit for the patient self reg / editing form

### ISSUE THREAD

**Issue by fzhao99** [CDCgov/prime-simplereport#4119]
Coming out of a design review w/ @kenieh of [4044](https://github.com/CDCgov/prime-simplereport/issues/4044), we noticed some content inconsistencies with the copy on the error states. cc @lauraykerr 

In certain situations, we alternate between "is missing" and "is missing or invalid" for empty form states

![Screen Shot 2022-08-02 at 11.46.44 AM.png](https://images.zenhubusercontent.com/6196b08a969160bed8994a93/1c1ea462-c82c-4b73-81f9-626f52873414)

The demographics form seems also to be missing error copy, which is an accessibility issue for colorblind and screen reader folks assuming the error bar isn't sufficient to communicate something's missing.

![Screen Shot 2022-08-02 at 11.47.57 AM.png](https://images.zenhubusercontent.com/6196b08a969160bed8994a93/415a6e50-e34f-47fa-8bf3-295c8afe73ef
)

**Comment by lauraykerr** [comment]
It looks like the self-registration form and add a new person forms have the same error messages, so updates should apply to both. Suggestions: 

1. First name: `First name is missing`

2. Last name: `Last name is missing`

3. Testing facility: `Testing facility is missing`

2. Date of birth: `Date of birth is missing, in the wrong format, or an invalid date`

2A-B. Date in future, too far in past: Leave as is

3. Primary phone number: Leave as is

4. Phone type: Leave as is

5. Street address: `Street address is missing`

6. State: `State is missing`

6. ZIP code: `ZIP code is missing` (doesn't appear to do any validation on numbers inputted) 

7. Race: `Race is missing`

8. Are you Hispanic or Latino?: `Ethnicity is missing` 

9. Sex assigned at birth: `Sex assigned at birth is missing`

**Comment by lauraykerr** [comment]
@emmastephenson Passing this back to engineering, thanks!
